### PR TITLE
Some formatting & environment spec edits

### PR DESCRIPTION
**Pull request by kofalt**
Some minor additions to the changes @wandell wrote earlier.

Largely inconsequential, some wrapping & line length edits, the only real changes are to the environment section, where I wanted to be extra clear about the environment provided.

It's probably easiest to review the [rich diff](https://github.com/flywheel-io/gears/pull/8/files?short_path=3abd632#diff-3abd6322bab97fc6b9c34e3f6d59da7e) which is cleaner to read.

### CODE REVIEW

File: spec/readme.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ This tar file can be created from most common container tools (e.g., Docker).
 
 ## The base folder
 
-To be a Flywheel gear, the container in the tar file must include a folder named: `/flywheel/v0`.
+To be a Flywheel gear, the container in the tar file must include a folder named: `/flywheel/v0`.<br>
```
Comment (gsfr) [inline spec/readme.md:13]: Having an explicit HTML tag in the markdown seems to go against the markdown intention. Can't we just add a blank line to get the same effect?

**Review by gsfr — COMMENTED** [review]
Changes look good, pending one small request.

File: spec/readme.md (hunk 1 above)
Comment (kofalt) [reply spec/readme.md:13]: Agreed that it's weird. I only ever use it for manual line breaks - purely a cosmetic change.
If only there were better ways:

* A blank line separating them would be interpreted as a new paragraph
* The [CommonMark spec on hard line breaks](http://spec.commonmark.org/0.27/#hard-line-breaks) offers two trailing spaces, but a lot of tools consider trailing whitespace to be an error - indeed, I have my editor set to automatically trim!
* The other option it offers is a trailing backslack - `foo\` - but github does not seem to have adopted CommonMark yet and that markup does not render correctly

So basically, I use `<br>` sparingly when the Github readme render would look nicer, with the assumption that we can switch to the non-HTML backslash when CommonMark takes over the world. That cool with you? :sunglasses:

Comment (kofalt) [reply spec/readme.md:13]: And just in case you thought markdown was consistent, even within a single vendor... a carriage return **in a comment** turns into a line break, but **in a repo file** will not! :roll_eyes:

**Review by gsfr — APPROVED** [review]
Good to go.
